terminus-ui/tabs/src/collection/tab-collection.component.ts

```diff
@@ -309,27 +315,6 @@ export class TsTabCollectionComponent implements AfterContentInit, AfterContentC
     this.focusChange.emit(this.createChangeEvent(index));
   }
 
-
```

Comment:
Did this code change have anything to do with linting, or just cleaning/organizing while you're in the file.

Reply:
We're now building these IDs directly in the template to remove function calls from the template.